tekton/release-pipeline.yaml

```diff
@@ -8,53 +8,149 @@ spec:
     - name: package
       description: package to release
       default: github.com/tektoncd/dashboard
+    - name: gitRevision
+      description: the git revision to release
     - name: imageRegistry
-      default: gcr.io/tekton-releases
+      description: The target image registry
+      default: gcr.io
+    - name: imageRegistryPath
+      description: The path (project) in the image registry
+      default: tekton-releases
     - name: versionTag
       description: The X.Y.Z version that the artifacts should be tagged with
+    - name: releaseBucket
+      description: >-
+                      bucket where the release is stored. The bucket must be
+                      project specific.
+      default: gs://tekton-releases-nightly/dashboard
     - name: releaseAsLatest
-      description: Whether to tag and publish this release as the latest
+      description: Whether to tag and publish release as Dashboard's latest
       default: "true"
-  resources:
-    - name: dashboard-source-repo
-      type: git
-    - name: bucket-for-dashboard
-      type: storage
-    - name: builtDashboardImage
-      type: image
+    - name: platforms
+      description: Platforms to  build images for (linux/amd64,linux/arm64 ...)
+      default: linux/amd64,linux/arm64,linux/s390x,linux/ppc64le
```

Comment:
same as above, should include `linux/arm` here to match existing releases by default